lua/autorun/laserlib.lua

```diff
@@ -268,16 +275,26 @@ local gtREFRACT = { -- https://en.wikipedia.org/wiki/List_of_refractive_indices
   ["models/props_combine/stasisshield_sheet"]   = {1.511, 0.427}  -- Blue temper glass
 }; gtREFRACT.Size = #gtREFRACT
 
+--[[
+ * Material configuration to use when override is missing
+ * Acts like a reference key jump for to the REFLECT set
+ * https://wiki.facepunch.com/gmod/Enums/MAT
+]]
 local gtMATYPE = {
-  [MAT_SNOW       ] = "white",
-  [MAT_GRATE      ] = "metal",
-  [MAT_CLIP       ] = "metal",
-  [MAT_METAL      ] = "metal",
-  [MAT_VENT       ] = "metal",
-  [MAT_GLASS      ] = "glass",
-  [MAT_WARPSHIELD ] = "glass"
+  [tostring(MAT_SNOW      )] = "white",
```

Comment:
`MAT_%` mecros have values in range `0:100` so Lua will allocate 128 element and fill just 4 of them when using keys as numbers